src/System.Linq/src/System/Linq/Errors.cs

```diff
@@ -6,18 +6,18 @@ namespace System.Linq
 {
     internal static class Error
     {
-        internal static Exception ArgumentNull(string s) { return new ArgumentNullException(s); }
+        internal static Exception ArgumentNull(string s) => new ArgumentNullException(s);
```

Comment:
I prefer it, though I prefer consistency more.  In this case there were only a few in the file, so I changed them all.